drops.go

```diff
@@ -0,0 +1,56 @@
+package helix
+
+// GetDropEntitlementsParams ...
+type GetDropEntitlementsParams struct {
+	EntitlementID string `query:"id"`
+	UserID        string `query:"user_id"`
+	GameID        string `query:"game_id"`
+	After         string `query:"after"`
+	First         int    `query:"first,20"` // Limit 100
+}
+
+// Entitlement ...
+type Entitlement struct {
+	EntitlementID string `json:"id"`
+	BenefitID     string `json:"benefit_id"`
```

Comment:
Please change `EntitlementID` to `ID` to be consistent with other structs in this library.

Reply:
Fixed this one as well 👍